server/src/sim.rs

```diff
@@ -30,8 +31,8 @@ use crate::postcard_helpers::{self, SaveEntity};
 
 pub struct Sim {
     cfg: Arc<SimConfig>,
-    rng: SmallRng,
     step: Step,
+    rng: SmallRng,
```

Comment:
To shrink the size of this diff, `rng` should be before `step` here and in the constructor below.